Description: Do not strip programs.
 Remove stripping options during installation.  This is handled by
 other means.
Author: Alberto Gonzalez Iniesta
Bug-Debian: http://bugs.debian.org/437618
Comment: interdiff netkit-telnet_0.17-{34,36}.diff
Forwarded: no
Last-Update: 2008-06-22

diff -u netkit-telnet-0.17/telnet/Makefile netkit-telnet-0.17/telnet/Makefile
--- netkit-telnet-0.17/telnet/Makefile
+++ netkit-telnet-0.17/telnet/Makefile
@@ -22,7 +22,7 @@
 	$(CXX) $(CXXFLAGS) -MM $(SRCS) >depend.mk
 
 install: telnet
-	install -s -m$(BINMODE) telnet $(INSTALLROOT)$(BINDIR)
+	install -m$(BINMODE) telnet $(INSTALLROOT)$(BINDIR)
 	install -m$(MANMODE) telnet.1 $(INSTALLROOT)$(MANDIR)/man1
 
 clean:
diff -u netkit-telnet-0.17/telnetd/Makefile netkit-telnet-0.17/telnetd/Makefile
--- netkit-telnet-0.17/telnetd/Makefile
+++ netkit-telnet-0.17/telnetd/Makefile
@@ -28,7 +28,7 @@
 telnetd.o: ../version.h
 
 install: telnetd
-	install -s -m$(DAEMONMODE) telnetd $(INSTALLROOT)$(SBINDIR)/in.telnetd
+	install -m$(DAEMONMODE) telnetd $(INSTALLROOT)$(SBINDIR)/in.telnetd
 	install -m$(MANMODE) issue.net.5 $(INSTALLROOT)$(MANDIR)/man5/
 	install -m$(MANMODE) telnetd.8 $(INSTALLROOT)$(MANDIR)/man8/in.telnetd.8
 	ln -sf in.telnetd.8 $(INSTALLROOT)$(MANDIR)/man8/telnetd.8
only in patch2:
unchanged:
--- netkit-telnet-0.17.orig/telnetlogin/Makefile
+++ netkit-telnet-0.17/telnetlogin/Makefile
@@ -11,7 +11,7 @@
 $(OBJS): ../version.h
 
 install: telnetlogin
-	install -s -m4750 -oroot -gtelnetd telnetlogin $(INSTALLROOT)$(SBINDIR)
+	install -m4750 -oroot -gtelnetd telnetlogin $(INSTALLROOT)$(SBINDIR)
 	install -m$(MANMODE) telnetlogin.8 $(INSTALLROOT)$(MANDIR)/man8
 
 clean:
